specification/dimensions/billingperiodend.md

```diff
@@ -0,0 +1,32 @@
+# Billing Period End
+
+Billing period represents the time window for which you have or will receive an invoice for. The time window is inclusive of the start date and exclusive of the end date.
+
+Billing Period End represents the end of a billing period.
+
+The BillingPeriodEnd column MUST be present in the billing data. This column MUST conform to [FOCUS Date/Time Requirements](#date/timerequirements) and MUST NOT contain null values. The sum of the Billed Cost metric for line items in a given billing period MUST match the total cost of the invoices received for that billing period.
```

Comment:
```suggestion
The BillingPeriodEnd column MUST be present in the billing data. This column MUST conform to [FOCUS Date/Time Requirements](#date/timerequirements) and MUST NOT contain null values. The sum of the billed cost metric for line items in a given billing period MUST match the total cost of the invoices received for that billing period.
```

Reply:
We refer to a column by its display name. We've gone back and forth on this multiple times but current plan is to go back to display name.